controllers/user.server.controller.js

```diff
@@ -36,10 +37,22 @@ exports.userLogout = function(req, res) {
  * @param res HTTP request response object
  */
 exports.userViewById = function(req, res) {
-    // TODO: implement userViewById()
-    res.json({ dummyTest: "userViewById() dummy test passes" });
+
```

Comment:
I suggest including some validation of the request data with a 400 "Bad request" returned if the data is invalid